backend/molgenis-emx2-semantics/src/main/java/org/molgenis/emx2/beaconv2/endpoints/individuals/ejp_rd_vp/EJP_VP_IndividualsQuery.java

```diff
@@ -156,6 +156,40 @@ else if (type.endsWith("SIO_010056")) {
         queryIndividuals(tables, filters.toArray(new String[0]));
 
     // only works because we only do AND queries, so one unmatched filter means no hit
+    List<String> removeIndividualIDs = removeIndividualIDs(ageQueries, resultSetsList);
```

Comment:
not sure why this needed to be changed but all tests are happy so all seems fine